.travis.yml

```diff
@@ -17,38 +17,48 @@
 language: go
 
 go:
-  - 1.13.x
-
+- 1.13.x
 env:
   global:
-    - GO111MODULE=on
-    - CC_TEST_REPORTER_ID=abdad8b2e2ec0bfdd7af57d955c44e9470f4d174e744b824047f3037800f5b40
-
+  - GO111MODULE=on
```

Comment:
do we still need this line since go mods are on by default since >1.11?

Reply:
not sure, I didn't change any other values aside from the notifications and then running travis lint but I can remove it if you think we don't need it anymore.

Reply:
yeah totally maybe try removing it? I think it should work still